modules/nodes-pods-priority-about.adoc

```diff
@@ -37,6 +38,8 @@ system-node-critical      2000001000   false            72m
 ** sdn-ovs
 ** sync
 
+* *openshift-user-critical* - Prometheus pods under `openshift-monitoring` namespace use `priorityClassName` `openshift-user-critical`. The monitoring workloads use `system-critical` as their first `priorityClass`.But it causes problem when monitoring uses excessive memory and the nodes can't evict them. Monitoring priority drops to give the scheduler flexibility, to move the heavy workloads around and keep critical nodes alive.
```

Comment:
Better changed as the following
* *openshift-user-critical* - The priority class is used with pods that are important for the cluster and can't bound their resource consumption, that is, have no predictable behavior of resource consumption. Prometheus pods under `openshift-monitoring` namespace and `openshift-user-workload-monitoring` namespace use `priorityClassName` `openshift-user-critical`. The monitoring workloads use `system-critical` as their first `priorityClass`.But it causes problem when monitoring uses excessive memory and the nodes can't evict them. Monitoring priority drops to give the scheduler flexibility, to move the heavy workloads around and keep critical nodes alive.

Reply:
acked